src/Microsoft.TestPlatform.ObjectModel/DataCollector/Events/DataCollectionEventArgs.cs

```diff
@@ -27,17 +27,9 @@ public DataCollectionEventArgs()
         /// Initializes the instance by storing the given information
         /// </summary>
         /// <param name="context">Context information for the event</param>
-        protected DataCollectionEventArgs(DataCollectionContext context) :
-            this(context, null)
+        protected DataCollectionEventArgs(DataCollectionContext context)
         {
-        }
-
-        protected DataCollectionEventArgs(DataCollectionContext context, Uri targetDataCollectorUri)
-        {
-            //EqtTrace.FailIf(context == null, "Context should not be null.");
-
             Context = context;
-            TargetDataCollectorUri = targetDataCollectorUri;
```

Comment:
Keep careful about this change, when you remove the property.

Reply:
@harshjain2 Searched for DataCollectors in devdiv using "TestPlatform.MSTest\ObjectModel\15.0\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll" to compile, they do not seem to be using this property.

https://devdiv.visualstudio.com/DevDiv/_search?type=code&lp=search-Project&text=TestPlatform.MSTest%5CObjectModel%5C15.0&result=DefaultCollection%2FDevDiv%2FVS%2FGBlab%2Fvsumain%2F%2Fsrc%2Fvset%2FQTools%2FExecution%2FAgent%2FPlugins%2FMediaRecorder%2FMediaRecorderV2%2FMediaRecorderV2.csproj&preview=0&filters=ProjectFilters%7BDevDiv%7DRepositoryFilters%7BVS%7DBranchFilters%7Blab%2Fvsumain%7D&_a=search
Remove if from Code, & lets complete the PR

Reply:
Done.